src/server/modelManager.js

```diff
@@ -1,5 +1,13 @@
+const Grid = require("./models/grid");
+
+var g;
+
 function createGrid(json)
 {
-
+    g=new Grid(json.nbKeel,json.nbFrame);
+    for (let i = 0; i < json.players.length; i++) {
```

Comment:
Maybe a forEach as follow can simplify the code
`array1.forEach(element => console.log(element));`

Reply:
done